### DISCUSSION TITLE
`register` with complex field name

### DISCUSSION THREAD

**Discussion by kevcao-certik** [react-hook-form/react-hook-form#9840] [Q&A] [answered]
What should I do if I have an object like this:
```json
{
    "foo": {
        "{\"bar\":\"baz\"}": <value>
    }
}
```

And I want to register `value`? Doing `register("foo.{\"bar\":\"baz\"}")` doesn't work and am getting an error. Is it possible to deal with these special characters?

**Comment by Moshyfawn** [comment, marked as answer]
RHF doesn't support special character in field names
